lambda-event-filtering/README.md

```diff
@@ -0,0 +1,22 @@
+# Cloud Pod — Lambda Event Filtering using SQS & DynamoDB
```

Comment:
@HarshCasper Let's also add the Launchpad badge here 🚀 - we can generate the link here: https://app.localstack.cloud/launchpad